Makefile

```diff
@@ -309,10 +327,20 @@ generate-go: $(CONTROLLER_GEN) $(CONVERSION_GEN) $(MOCKGEN) $(DEFAULTER_GEN) ##
 		--input-dirs=./api/v1alpha2 \
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
-	go generate ./...
+
+.PHONY: generate-go-eks-bootstrap
+generate-go-eks-bootstrap: $(CONTROLLER_GEN) $(CONVERSION_GEN)
+	$(CONTROLLER_GEN) \
+		paths=./bootstrap/eks/api/... \
+		paths=./bootstrap/eks/types/... \
+		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt
 
 .PHONY: generate-manifests
-generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
```

Comment:
I don't think deleting all this line was your real intention:
```
make[1]: Nothing to be done for `generate-manifests'.
```

Reply:
ah definitely not! will fix